src/hazelcore/Action_Exp.re

```diff
@@ -3458,3 +3476,25 @@ and ana_perform_subsume =
       }
     }
   };
+
+let syn_perform =
```

Comment:
this is from that conversation we had about adding contracts into the codebase. probably shouldn't be in this PR.

Reply:
why not? isn't this a reasonable check to have?